src/components/examiner-custom-login.js

```diff
@@ -22,33 +23,61 @@ function ExaminerCustomLoginComponent() {
 
   const { loginExaminer } = useAppContext();
 
-  // TODO - call this when submit
-  const handleLogIn = (email, password) => {
-    loginExaminer({ email, password }, true);
+  const initialValues = {
+    email: "",
+    password: "",
   };
 
+  const loginDetailsSchema = Yup.object().shape({
+    email: Yup.string().email("Invalid email").required("Email is required"),
+    password: Yup.string()
+      .required("Password is required")
+      .min(8, "Password is too short - should be 8 chars minimum.")
+      .max(20, "Password is too long - should be 20 chars maximum.")
+      .matches(
+        /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,})/,
+        "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and One Special Case Character"
+      ),
+  });
+
+  const onSubmit = (values) => {
+    loginExaminer(values);
```

Comment:
Can we change this as follows.
```
loginExaminer(values, true);
```
As the function definition in https://github.com/cse-mosip/Exam-Attendance-UI/blob/main/src/context/appContext.js , if we are using loginExaminer with email, we need to make withEmail=true.

Reply:
Done